chapter02/axp/ex01

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env perl
+
+use 5.014;
+use warnings;
```

Comment:
Little known fact: `use strict` is unnecessary for any versions >= 5.012.

> Similarly, if the specified Perl version is greater than or equal to 5.12.0, strictures are enabled lexically as with use strict 
http://perldoc.perl.org/functions/use.html